### DISCUSSION TITLE
《科學》雜誌評選2021年重大科技突破：「人工智能驅動的蛋白質預測」

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#216] [General]
https://www.storm.mg/article/4130476
《科學》稱這項技術的革命性可以同此前的基因編輯技術CRISPR/Cas9相媲美。借助基因編輯技術可以輕易而精准地進行基因改變，目前已成為分子生物學中的標准工具。

**Comment by handsomeniko** [comment]
AI其實對於生科方面有很大的貢獻，我相信未來多多少少各領域都會與AI有所關聯。

**Comment by ghost** [comment]
人類的醫學一天一天的在進步 也是多虧了AI的幫忙 讓我覺得AI是一個很強大的存在

**Comment by Shiba010** [comment]
最近也了解到AI在醫學上的應用，尤其在辨識和預測方面越來越厲害了！

**Comment by lx3211** [comment]
感覺以後AI在醫學裡會扮演很重要的角色

**Comment by NcuJerryKao** [comment]
AI若將來能 改變人類基因 植入抗癌DNA(或是抗中保台DNA)  將會對人類有不少幫助 或許真能讓癌症消失
不過又有另一個問題 死亡人口大幅減少 那總人口提升太快 要想辦法產出糧食
將會是未來的課題

**Comment by BingRong1217** [comment]
AI結合生物科技可以對我們的生活有很大幫助

**Comment by Yilin20021016** [comment]
感覺有點像AI報告的熱量記錄阿

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Raywei0214** [comment]
就我個人來說，蛋白質預測對我的意義，不能不說非常重大。領悟其中的道理也不是那麼的困難。薩特說過一句發人省思的話，如果試圖改變一些東西，首先應該接受許多東西。這段話讓我的心境提高了一個層次。

**Comment by RogelioKG** [comment]
這應該是 AlphaFold 技術吧。
我記得他們的模型好像不擅長預測那種有配體的蛋白質的結構，
像高中生物課本提到的血紅素，
中心的那個血基質 (有卟啉環然後螯合住鐵離子的那個) 就是配體。
這種配體經常影響整個蛋白質的立體構型。
這項技術應該是還有再繼續成長的空間。

**Comment by larry40305** [comment]
透過AI能幫助我們更快速辨識一些人類無法輕易辨識的細節。

**Comment by ghost** [comment]
在這個時代，基本上各行各業都可以透過ai得到更快速的進步，特別是生科產業，畢竟生物是非常複雜的東西，如果能使用ai就能比人眼研究的更加透徹。

**Comment by bww0401** [comment]
我認為AI在醫學真的幫助很多，像是血糖檢測、超音波影像判讀等等，若未來發展成熟並普及的話，將有效解決台灣醫生不夠的問題，希望也能改善醫療資源分配不均的問題

**Comment by skyling0299** [comment]
對生物化學、細胞生物學、遺傳發育、神經生物學、微生物學、病理藥理等一大批生命學科和研究領域來說，這會大大改進人們對於生命過程的理解。比如，遺傳學家也許積累了大量數據，但如果不知道蛋白質結構，就沒法研究某個突變對於蛋白功能的影響。現在不同了，通過AlphaFold2的結構預測就能查看人類遺傳病中的每一個突變在相關蛋白結構裡的具體位置，進而有可能推測出蛋白功能如何受到影響。

**Comment by skyling0299** [comment]
對生物化學、細胞生物學、遺傳發育、神經生物學、微生物學、病理藥理等一大批生命學科和研究領域來說，這會大大改進人們對於生命過程的理解。比如，遺傳學家也許積累了大量數據，但如果不知道蛋白質結構，就沒法研究某個突變對於蛋白功能的影響。現在不同了，通過AlphaFold2的結構預測就能查看人類遺傳病中的每一個突變在相關蛋白結構裡的具體位置，進而有可能推測出蛋白功能如何受到影響。
